backend/src/core/routes/index.ts

```diff
@@ -170,7 +170,9 @@ export const InitV1Route = (app: Application): Router => {
    *     bearerAuth:
    *       type: http
    *       scheme: bearer
+   *       bearerFormat: username_versionNumber_apiKey
    *     cookieAuth:
+   *       description: while this description (is correct)[https://swagger.io/docs/specification/authentication/cookie-authentication/], do note that cookie authentication is currently not supported for "try it out" requests due to browser security restrictions.
```

Comment:
It is supported though, we're just using the wrong cookie name. Should be `postmangovsg-v2`
![Screenshot 2022-10-02 at 12 17 55 PM](https://user-images.githubusercontent.com/12974927/193437616-f0edf6ca-147e-474f-9792-23260b3eb65e.png)

Reply:
It was working when I "tried it out"

Reply:
Ohh I copied that line from the page you linked and it didn't work when I tried. 

When I checked the cookie name from my browser, it didn't have the v2, but let me try again when I have the time!

Reply:
where did you get the `COOKIE_NAME` in the screenshot from? mine is here:

<img width="780" alt="image" src="https://user-images.githubusercontent.com/67887489/194741580-a2665f77-7120-4b0e-b0fa-2f4807b99fbc.png">

and I tried it here (which didn't work):

![image](https://user-images.githubusercontent.com/67887489/194741626-3b5be114-b111-41fa-9208-5ba45dec0868.png)

Reply:
oh I found it on production, that one has the `v2`

Reply:
hmm but trying out production using that cookie still gives me a Internal Server Error 
<img width="1130" alt="image" src="https://user-images.githubusercontent.com/67887489/194741769-a241045d-8258-457b-8842-41518067c4fe.png">

Reply:
from the website [you linked](https://swagger.io/docs/specification/authentication/cookie-authentication/) 
![image](https://user-images.githubusercontent.com/67887489/194741813-4effcc4b-1e4b-45f7-99da-29fccdce48c9.png)

Reply:
I guess we can remove it then. We're narrowing this swagger doc to just API later on anw so

Reply:
by remove, you mean remove the `cookieAuth` right? will do

Reply:
done